web-dev/src/components/DetailScreen/similar-item.js

```diff
@@ -0,0 +1,18 @@
+import {Link} from "react-router-dom";
+import React from "react";
+import "./detail-screen.css";
+
+const SimilarItem = ({item = {}}) => {
```

Comment:
What is the similar item?

Reply:
It is a new section I added to the detail page. The similar items are elements of similar titles section. It shows movie/title that is similar to the one in the detail page. The API provides that data in the query, so I take advantage of it